services/ui-src/src/components/modal/Modal.tsx

```diff
@@ -0,0 +1,173 @@
+import React from "react";
+
+// components
+import {
+  Box,
+  Button,
+  Heading,
+  Modal as ChakraModal,
+  ModalBody,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  Text,
+} from "@chakra-ui/react";
+import { CloseIcon } from "@cmsgov/design-system";
+import { makeMediaQueryClasses } from "utils";
+
+export const Modal = ({ actionFunction, content, modalState }: Props) => {
+  const mqClasses = makeMediaQueryClasses();
+  return (
+    <ChakraModal isOpen={modalState.isOpen} onClose={modalState.onClose}>
+      <ModalOverlay />
+      <ModalContent sx={sx.modalContent}>
+        <ModalHeader sx={sx.modalHeader}>
+          <Heading as="h1" sx={sx.modalHeaderText}>
+            {content.heading}
+          </Heading>
+        </ModalHeader>
+        <Box sx={sx.modalCloseContainer}>
```

Comment:
Since this is styled with `display: flex` you could just use the `<Flex />` component here instead of `<Box />`.